OpenRA.Game/Graphics/ChromeProvider.cs

```diff
@@ -54,10 +54,10 @@ public class Collection
 
 		public static IReadOnlyDictionary<string, Collection> Collections { get; private set; }
 		static Dictionary<string, Collection> collections;
-		static Dictionary<string, Pair<Sheet, int>> cachedSheets;
+		static Dictionary<string, (Sheet Sheet, int density)> cachedSheets;
```

Comment:
If I understand correctly that these are being treated as public members of an instance so are in PascalCase, I think `density` wants to be `Density`:
```suggestion
		static Dictionary<string, (Sheet Sheet, int Density)> cachedSheets;
```